src/_layouts/default.html

```diff
@@ -31,8 +31,6 @@
       {% include page-content.html %}
     </main>
     {% include contribute.html %}
-    <script src="/assets/js/plausible.js">
-      plausible('pageview', {u: prepareUrl([])})
-    </script>
+    <script src="/assets/js/plausible.js"></script>
```

Comment:
Ah - I was wondering about this in the previous commit. Does it need to be a separate commit or could it just be squashed into the last?